packages/kit/src/runtime/load.js

```diff
@@ -5,9 +5,156 @@
 export function normalize(loaded) {
 	// TODO should this behaviour be dev-only?
 
-	if (loaded.error) {
-		const error = typeof loaded.error === 'string' ? new Error(loaded.error) : loaded.error;
+	if (
+		loaded.error ||
+		(loaded.status && loaded.status >= 400 && loaded.status <= 599 && !loaded.redirect)
+	) {
 		const status = loaded.status;
+		if (!loaded.error) {
+			let errorMessage = 'not assigned';
+			switch (status) {
+				case 400:
```

Comment:
There was some discussion of whether we should include the `http-status-codes` library (https://github.com/sveltejs/kit/issues/1439). I didn't think it was worth including just to make it available to users. But if we're actually going to be using it in SvelteKit's own code, I might reconsider rather than having this list here. I'm not sure if @Conduitry or Rich have other thoughts about it

Reply:
In 1ccc6f33d25b56f0966a08bb2db67c749e898dea there are only these codes 400, 401, 403, 404, 409, 500 which are the most common. We could probably support this lesser list and either send a 500 or pass the correct status code with unspecified as the error text i.e. {status: 402, error:"Unspecified"}